base/file.jl

```diff
@@ -481,9 +497,6 @@ function tempname()
     return s
 end
```

Comment:
Change to use `tempdir()` for `d`?

Reply:
Will make these changes in a separate PR.

Reply:
sgtm. feel free to merge.